css/main.css

```diff
@@ -234,22 +208,14 @@ input {
 	border-radius: 6px;
 }
 
-.hero {
-	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("https://bit.ly/2N5kvOq");
-	height: 400px;
-	background-position: center;
-	background-repeat: no-repeat;
-	background-size: cover;
-	position: relative;
-}
-
 .hero-text {
```

Comment:
this is _exactly_ the same as `.ingredients_text`... no need to duplicate... just add make one class for both.

Reply:
Resolved -#590c084